js/pages/profiles/profile-manager.js

```diff
@@ -309,6 +311,8 @@ define([
 				if (this.personId()) {
 					this.loadPerson();
 				}
+
+				this.plugins = pluginRegistry.findByType('atlas-profile-widget');
```

Comment:
Should we establish a set of constants for each plugin type we support?  so cohort reports and profiles are the two types I see so far...